src/components/GlobalPreferences/Notifications/ManageNotifications.js

```diff
@@ -0,0 +1,384 @@
+    const onOrganizationChange = useCallback(
+      idx => {
+        setSelectedOrganization(idx)
+      },
+      [setSelectedOrganization]
+    )
+
+    // Get unique app names by matching subscriptions with
+    const subscriptionApps = Array.from(
+      new Set(subscriptions.map(subscription => subscription.appName))
+    )
+    const [selectedApp, setSelectedApp] = useState(-1)
+    const onAppChange = useCallback(
+      idx => {
+        setSelectedApp(idx)
+      },
+      [setSelectedApp]
+    )
+    const events = Array.from(
+      new Set(subscriptions.map(subscription => subscription.eventName))
+    )
+    const [selectedEvent, setSelectedEvent] = useState(-1)
+    const onEventChange = useCallback(
+      idx => {
+        setSelectedEvent(idx)
+      },
+      [setSelectedEvent]
+    )
+    const onClearFilters = useCallback(() => {
+      setSelectedEvent(-1)
+      setSelectedApp(-1)
+      setSelectedOrganization(-1)
+    }, [setSelectedEvent, setSelectedApp, setSelectedOrganization])
+
+    const filteredSubscriptions = filterSubscriptions({
+      subscriptions,
+      event: events[selectedEvent],
+      appName: subscriptionApps[selectedApp],
+      organization: organizations[selectedOrganization],
+    })
+    const theme = useTheme()
+    return (
+      <DataView
+        fields={[
+          {
+            label: 'Organization',
+            priority: 3,
+          },
+          {
+            label: 'App',
+            priority: 2,
+          },
+          {
+            label: 'Contract',
+            priority: 2,
+          },
+          {
+            label: 'Event',
+            priority: 1,
+          },
+        ]}
+        heading={
+          <React.Fragment>
+            <div
+              css={`
+                height: ${9 * GU}px;
+                display: flex;
+                align-items: center;
+                justify-content: space-between;
+              `}
+            >
+              <div
+                css={`
+                  color: ${theme.content};
+                  ${textStyle('body1')}
+                `}
+              >
+                Subscriptions
+              </div>
+              {selectedSubscriptions.length > 0 && (
+                <div css="text-align: right;">
+                  <Button disabled={false} onClick={handleUnsubscribe}>
+                    {isSubmitting ? (
+                      <LoadingRing
+                        css={`
+                          margin-right: ${GU}px;
+                        `}
+                      />
+                    ) : (
+                      <IconMail
+                        css={`
+                          color: ${theme.negative};
+                          margin-right: ${GU}px;
+                        `}
+                      />
+                    )}{' '}
+                    Unsubscribe
+                  </Button>
+                </div>
+              )}
+            </div>
+            <SubscriptionFilters
```

Comment:
@dizzypaty for other sections with filters we have compact views that use icons, do we have a similar thing here?

![image](https://user-images.githubusercontent.com/3072458/63695014-029b1900-c818-11e9-92a6-94887e2e0bd3.png)

Reply:
eg:
![image](https://user-images.githubusercontent.com/3072458/63695056-1e9eba80-c818-11e9-9af1-d36e8efc03a0.png)